pkg/middleware/logging/logging.go

```diff
@@ -78,18 +80,22 @@ func (t *logTripper) Request(r *http.Request) (*http.Response, error) {
 	if err != nil {
 		if fragment != nil {
 			fmt.Println(err)
```

Comment:
Hmm, although it isn't related to this PR, perhaps we should drop this line completely and always use `t.logger`:

```suggestion
```